config.py

```diff
@@ -13,6 +13,7 @@
 UVICORN_UDS = config("UVICORN_UDS", default=None)
 UVICORN_SSL_CERTFILE = config("UVICORN_SSL_CERTFILE", default=None)
 UVICORN_SSL_KEYFILE = config("UVICORN_SSL_KEYFILE", default=None)
+UVICORN_SSL_CA_TYPE = config("UVICORN_SSL_CA_TYPE", default="public").lower()
```

Comment:
What do you mean?

Why do we need a default in config.py?

Or why do we need CA_TYPE at all?

The default is to avoid specifying it in .env variables, it will always be public and CA_TYPE will always be validated

It is necessary to change CA type for test environments, for example dev/test - where self-signed certificates are used. This is extremely useful for testing large installations, with a large number of devices spread over different availability zones and so on.

For example, for quick deployment I have a set of ansible playbooks that deploy a marzban installation in 5-8 minutes, with a domain, certificates, nodes and so on.

In such a situation self-signed certificates are very applicable.